gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/AltTextSection.kt

```diff
@@ -0,0 +1,174 @@
+import java.net.URI
+import java.net.URL
+
+@Composable
+internal fun AltTextSection(
+    state: MainContentSection.AltTextUiState,
+    onAltTextChanged: (String) -> Unit,
+    onWhatIsAltTextClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val sectionPadding = 16.dp
+    var altText by remember { mutableStateOf(state.avatar.altText) }
+    Surface(modifier = modifier.fillMaxWidth()) {
+        Box(
+            modifier = Modifier
+                .border(
+                    width = 1.dp,
+                    color = MaterialTheme.colorScheme.surfaceContainerHighest,
+                    shape = RoundedCornerShape(8.dp),
+                )
+                .padding(sectionPadding),
+        ) {
+            Column {
+                Row(
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween,
+                    modifier = Modifier.fillMaxWidth(),
+                ) {
+                    QESectionTitle(
+                        title = stringResource(R.string.gravatar_qe_avatar_alt_text_section_title),
+                        modifier = Modifier,
+                    )
+                    Text(
+                        text = stringResource(id = R.string.gravatar_qe_avatar_alt_text_section_what_is),
+                        color = MaterialTheme.colorScheme.primary,
+                        modifier = Modifier.clickable { onWhatIsAltTextClicked() },
+                    )
+                }
+
+                Row(
+                    verticalAlignment = Alignment.Top,
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 16.dp),
+                ) {
+                    val cornerRadius = 8.dp
+                    val avatarSize = 96.dp
+                    val sizePx = with(LocalDensity.current) { avatarSize.roundToPx() }
+                    AsyncImage(
+                        model = state.avatar.imageUrlWithSize(sizePx),
+                        contentDescription = stringResource(
+                            id = R.string.gravatar_qe_selectable_avatar_content_description,
+                        ),
+                        modifier = Modifier
+                            .size(96.dp)
+                            .border(
+                                1.dp,
+                                MaterialTheme.colorScheme.surfaceDim,
+                                RoundedCornerShape(cornerRadius),
+                            )
+                            .clip(RoundedCornerShape(cornerRadius)),
+                    )
+                    val placeholder = stringResource(R.string.gravatar_qe_avatar_alt_text_section_placeholder)
+                    /*
+                     * TextField is still Experimental so using this workaround to avoid its use.
+                     * This needs to be improved.
+                     */
+                    BasicTextField(
```

Comment:
`TextField` is experimental, so we need to use `BasicTextField`. The major problem with this composable is the "placeholder". It enabled the placeholder by using:

```
decorator = TextFieldDefaults.TextFieldDecorationBox(
```

Which is also experimental, so this is a pain for us.

I've implemented a workaround but also created a [task](https://github.com/Automattic/Gravatar-SDK-Android/issues/499) to improve this.

Current behavior:

https://github.com/user-attachments/assets/e57aa03a-90f9-4e5a-85be-9dd81f5cf450

Reply:
Hmm this doesn't work at all like a placeholder, right? I mean, when I tap on the placeholder it will place the cursor wherever I tap not only in the beginning.

Reply:
Exactly, that's the problem. If you prefer, we can remove this and work on it in the subtask.

Reply:
Yeah, I think this is no better than no placeholder so maybe makes sense to not have it.